requirements.txt

```diff
@@ -1,15 +1,42 @@
+################################################
+## DO NOT EDIT - AUTOMATICALLY GENERATED FILE ##
+################################################
+astroid==1.4.7
+backports.ssl-match-hostname==3.5.0.1
+git+https://github.com/CDECatapult/behave.git@1.3.1
```

Comment:
Problem with the script. This should be
+git+https://github.com/CDECatapult/behave.git@v1.3.1